x/ecocredit/core/utils.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/regen-network/regen-ledger/x/ecocredit"
 )
 
+var errBadReq = sdkerrors.ErrInvalidRequest
+
```

Comment:
Is this necessary? I would prefer to see `sdkerrors.ErrInvalidRequest` used directly.

Reply:
shorter lines are easier to read and don't require to break lines.

Reply:
I agree. Shorter lines are easier to read.